src/refinement/ptq.jl

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    PTQ()
+
+Refinement of triangle meshes.
+A triangle is subdivided into four triangles.
```

Comment:
Please try to follow the docstring convention of other methods. In particular, it is very helpful when we provide references in the literature. Is there a published paper discussing/using this method?

Reply:
The CGAL documentation refers to a master thesis:

Charles Loop. *Smooth subdivision surfaces based on triangles. Master's thesis, University of Utah, 1987.*

Lol, I thought that the world "loop" was refering to a for-loop...

Reply:
Awesome, we should definitely add this his thesis to the docstring.